UncomplicatedCustomRoles/UncomplicatedCustomRoles.csproj

```diff
@@ -462,12 +462,19 @@
     <Reference Include="zxing, Version=0.16.9.0, Culture=neutral, PublicKeyToken=4e88037ac681fe60, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>$(EXILED_REFERENCES)\zxing.dll</HintPath>
+    </Reference>
+	<Reference Include="RespawnTimer, Version=0.0.0.0, Culture=neutral, Culture=neutral, processorArchitecture=MSIL">

```

Comment:
we can't do that as the plugin will require the `RespawnTimer.dll` in order to work

Reply:
Yea this is why i put it in an other plugin.